### DISCUSSION TITLE
[icon] 使用icon功能需要外网权限？

### DISCUSSION THREAD

**Discussion by yanjao** [Tencent/tdesign-vue#747] [General]
### 这个功能解决了什么问题

需要请求https://tdesign.gtimg.com/icon/0.0.3/fonts/index.js，内网环境下不可以使用。

### 你建议的方案是什么

不使用外网访问可以放在包内吗？

**Comment by uyarn** [comment]
有两种处理方式
1. 使用按需引入的方式 不要直接用t-icon t-icon需要加载svgsprite资源 参考https://tdesign.tencent.com/vue/components/icon#svg-%E6%8C%89%E9%9C%80%E5%BC%95%E5%85%A5
2. 将https://tdesign.gtimg.com/icon/0.0.3/fonts/index.js 部署到你可以访问的地址 然后用自定义URL的方式引入 参考https://tdesign.tencent.com/vue/components/icon#svg-%E9%AB%98%E7%BA%A7%E7%94%A8%E6%B3%95
